mungegithub/github/github.go

```diff
@@ -1347,17 +1381,19 @@ func (obj *MungeObject) GetFileContents(file, sha string) (string, error) {
 
 // MergeCommit will return the sha of the merge. PRs which have not merged
 // (or if we hit an error) will return nil
-func (obj *MungeObject) MergeCommit() *string {
+//CHECK-UPDATED
+func (obj *MungeObject) MergeCommit() (*string, error) {
 	events, err := obj.GetEvents()
 	if err != nil {
-		return nil
+		return nil, err
+		glog.Errorf("Error: %v", err)
```

Comment:
IMO if you're returning an error, it's bad form to also glog it. One or the other.